tests/load/gcp/lma.tpl.conf

```
@@ -0,0 +1,133 @@
+#cloud-config
+
+write_files:
+- path: /run/prom-ingress.yaml
+  content: |
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: prom-ingress
+      namespace: lma-load-test
+      annotations:
+        nginx.ingress.kubernetes.io/use-regex: "true"
+        nginx.ingress.kubernetes.io/rewrite-target: /$1
+    spec:
+      ingressClassName: public
+      rules:
+      - http:
+          paths:
+          - path: /prom/(.*)
+            pathType: Prefix
+            backend:
+              service:
+                name: prometheus
+                port:
+                  number: 9090
+          - path: /loki/(.*)
+            pathType: Prefix
+            backend:
+              service:
+                name: loki
+                port:
+                  number: 3100
+- path: /run/overlay-load-test.yaml
+  content: |
+    --- # overlay.yaml
+    applications:
+      scrape-target:
+        charm: prometheus-scrape-target-k8s
+        scale: 1
+        trust: true
+        channel: edge
+        options:
+          targets: ${join(",", [for port in PORTS: "${AVALANCHE_URL}:${port}"])}
+      scrape-config:
+        charm: prometheus-scrape-config-k8s
+        scale: 1
+        trust: true
+        channel: edge
+        options:
+          scrape-interval: ${SCRAPE_INTERVAL}s
+          scrape-timeout: ${max(SCRAPE_INTERVAL - 1, 1)}s
+
+    relations:
+    - - scrape-target:metrics-endpoint
+      - scrape-config:configurable-scrape-jobs
+    - - scrape-config:metrics-endpoint
+      - prometheus:metrics-endpoint
+
+package_update: true
+
+packages:
+- python3-pip
+- jq
+
+snap:
+  commands:
+  - snap install --classic juju
+  - snap install --classic microk8s
+  - snap alias microk8s.kubectl kubectl
+
+runcmd:
+- |
+  # disable swap
+  sysctl vm.swappiness=0
+  echo "vm.swappiness = 0" | tee -a /etc/sysctl.conf
+
+- |
+  # disable unnecessary services
+  systemctl disable man-db.timer man-db.service --now
+  systemctl disable apport.service apport-autoreport.service  --now
+  systemctl disable apt-daily.service apt-daily.timer --now
+  systemctl disable apt-daily-upgrade.service apt-daily-upgrade.timer --now
+  systemctl disable unattended-upgrades.service --now
+  systemctl disable motd-news.service motd-news.timer --now
+  systemctl disable bluetooth.target --now
+  systemctl disable ua-messaging.service ua-messaging.timer --now
+  systemctl disable systemd-tmpfiles-clean.timer --now
+
+- |
+  # setup microk8s and bootstrap
```

```suggestion
  set -euo pipefail
  # setup microk8s and bootstrap
```

See below comment for context.